assemblies/evaluating-rag-systems.adoc

```diff
@@ -0,0 +1,103 @@
+
+Ragas is integrated with {productname-short} through the Llama Stack evaluation API and supports two deployment modes: an inline provider for development and testing, and a remote provider for production-scale evaluations using {productname-short} pipelines.
+
+== About Ragas evaluation
+
+Ragas addresses the unique challenges of evaluating RAG systems by providing metrics that assess both the retrieval and generation components of your application. Unlike traditional language model evaluation that focuses solely on output quality, Ragas evaluates how well your system retrieves relevant context and generates responses grounded in that context.
+
+=== Key Ragas metrics
+
+Ragas provides link:https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/[multiple metrics] for evaluating RAG systems. Here are some of the metrics:
+
+*Faithfulness*:: Measures how factually consistent the generated answer is with the retrieved context. A high faithfulness score indicates that the answer is well-grounded in the source documents, reducing the risk of hallucinations. This is critical for enterprise and regulated environments where accuracy and trustworthiness are paramount.
+
+*Answer Relevancy*:: Evaluates how relevant the generated answer is to the input question. This metric ensures that your RAG system provides pertinent responses rather than generic or off-topic information.
+
+*Context Precision*:: Measures the precision of the retrieval component by evaluating whether the retrieved context chunks contain information relevant to answering the question. High precision indicates that your retrieval system is returning focused, relevant documents rather than irrelevant noise.
+
+*Context Recall*:: Measures the recall of the retrieval component by evaluating whether all necessary information for answering the question is present in the retrieved contexts. High recall ensures that your retrieval system is not missing important information.
+
+*Answer Correctness*:: Compares the generated answer with a ground truth reference answer to measure accuracy. This metric is useful when you have labeled evaluation datasets with known correct answers.
+
+*Answer Similarity*:: Measures the semantic similarity between the generated answer and a reference answer, providing a more nuanced assessment than exact string matching.
+
+=== Use cases for Ragas in AI engineering workflows
+
+Ragas enables AI engineers to accomplish the following tasks:
+
+*Automate quality checks*::
+Create reproducible, objective evaluation jobs that can be automatically triggered after every code commit or model update. This establishes quality gates to prevent regressions and ensures that only high-quality RAG configurations are deployed to production.
```

Comment:
1) identify what you mean by "This"
for example "This task"
or because "establishes quality gates" is hard to parse, consider:
"These automated checks prevent regressions.."

2) avoid passive "are deployed"
consider:
...ensures that you deploy only high-quality configurations for production.